src/components/Link/linkExtensions.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+/**
+ * A HOC that provides a bit more intelligence / intuitive
+ * function to the React-Router Link
+ */
+
+const isExternal = url => /^(https?:)*\/\//.test(url);
```

Comment:
This is not actually an external url, this is an absolute one. It is possible to provide an internal URL with an absolute path. See https://stackoverflow.com/questions/904046/absolute-urls-relative-urls-and/904066#904066

Reply:
While possible, seems unlikely based on our usage to date. But I could see an argument to remove heuristics and make the `external` prop explicit from the user. Opinions?

Reply:
I suppose this method is okay if we are using it purely for styling.